nshlib/Kconfig

```
@@ -242,6 +242,14 @@ config NSH_BUILTIN_APPS
 		more information).  This options requires support for builtin
 		applications (BUILTIN).
 
+config NSH_BUILTIN_AS_COMMAND
+	bool "run builtin apps as command"
+	default n
+	depends on BUILTIN
+	---help---
+		set if users want to run builtin app directly without creating a
+		separate thread
+
```

```suggestion
config NSH_BUILTIN_AS_COMMAND
	bool "Run builtin apps as command"
	default n
	depends on BUILTIN
	---help---
		Enable to run builtin application directly without creating a separate thread.

```
Spelling fixes please.